lib/tasks/user_stats.rake

```diff
@@ -0,0 +1,10 @@
+task user_stats: 'kitsu:user_stats_setup'
```

Comment:
oh I failed at copying the `stats.rake` :D. it should be `user_stats_setup` (like 2 lines down)